src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,33 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
+        LocalDate localDateFrom = LocalDate.parse(dateFrom, formatter);
+        LocalDate localDateTo = LocalDate.parse(dateTo, formatter);
+        StringBuilder stringBuilder = new StringBuilder("Report for period " + dateFrom + " - " + dateTo);
```

Comment:
```suggestion
        StringBuilder report = new StringBuilder("Report for period " + dateFrom + " - " + dateTo);
```
неймінг краще обирати відносно призначення змінної а не її типу